src/commands/codepush/lib/release-command-skeleton.ts

```diff
@@ -81,31 +106,25 @@ export default class CodePushReleaseCommandSkeleton extends AppCommand {
     const updateContentsZipPath = await zip(this.updateContentsPath);
 
     try {
-      const httpRequest = await out.progress("Creating CodePush release...", clientRequest<models.CodePushRelease>(
-        (cb) => client.codePushDeploymentReleases.create(
-          this.deploymentName,
-          this.targetBinaryVersion,
-          this.app.ownerName,
-          this.app.appName,
-          {
-            packageParameter: fs.createReadStream(updateContentsZipPath),
-            deploymentName1: this.deploymentName,
-            description: this.description,
-            disabled: this.disabled, 
-            mandatory: this.mandatory, 
-            noDuplicateReleaseError: this.noDuplicateReleaseError,
-            rollout: this.rollout,
-          },
-          cb)));
-
+      var user = await getUser();
+      const serverUrl = environments(this.environmentName || user.environment).managementEndpoint;
+      
+      await this.releaseStrategy.release(client, this.app, this.deploymentName, updateContentsZipPath, {
```

Comment:
Shouldn't this be wrapped in an `out.progress` call?

Reply:
Got it, will do